### PR TITLE
Release automation  - Publish GitHub release notes

### PR DESCRIPTION
**Pull request by andoni-guzman**
This workflows is part of the release automation process, this workflows do the same that publish_github_release_notes.sh but now the release manager doesn't need to interact, only run the workflow and insert the release version as parameter of the workflows 
1 Filed added 
- .github/workflows/publish_github_release_notes.yml
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: .github/workflows/publish_github_release_notes.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      RELEASE_VERSION: ${{ github.event.inputs.RELEASE_VERSION}}
+    name: Publish Github Release Notes
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Publish github release notes
+        run: |
+          POST_PATH=$(echo "website/www/site/content/en/blog/beam-${{env.RELEASE_VERSION}}.md")
```
Comment (damccorm) [inline .github/workflows/publish_github_release_notes.yml:55]:
```suggestion
          POST_PATH="website/www/site/content/en/blog/beam-${{env.RELEASE_VERSION}}.md"
```

Any reason to nest this in an echo?

**Review by damccorm — COMMENTED** [review]
This generally LGTM, though the sed blocks are pretty tough to review

File: .github/workflows/publish_github_release_notes.yml (hunk 1 above)
Comment (andoni-guzman) [reply .github/workflows/publish_github_release_notes.yml]: Not really, deleted in the last commit

**Review by damccorm — APPROVED** [review]
LGTM

**Pull request merged**